src/components/fieldset/address-wrapper.scss

```diff
@@ -0,0 +1,9 @@
+@import "@govuk-frontend/globals/common";
+
+// This is a separate stylesheet for the address example to limit the width of the fieldset,  simulating the content being in a two-thirds column (most common scenario that the code will be pasted) without adding it to the markup.
```

Comment:
Can we wrap these comments at 80 characters? Like [this](https://github.com/alphagov/govuk-design-system/blob/master/src/stylesheets/components/_header.scss#L1-L9).